packages/webapp/src/app/page.tsx

```diff
@@ -1,8 +1,9 @@
 import Link from 'next/link';
 
-export default function Home() {
+export default  function Home() {
   const webappUrl = process.env.WEBAPP_URL || 'https://t.me/StarBeamBot/wallet';
 
+
   return (
     <div className="grid grid-rows-[20px_1fr_20px] items-center justify-items-center min-h-screen p-8 pb-20 gap-16 sm:p-20 font-[family-name:var(--font-geist-sans)]">
```

Comment:
Just random formatting changes, seems out of scope.

Reply:
I perfomed PoC there initially before i removed it